src/StatusQ/Buttons/StatusIconButton.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+
+Rectangle {
+    id: statusIconButton
+
+    property color iconNormalColor: "transparent"
+    property color iconDisabledColor: "transparent"
+
+    property color backgroundNormalColor: "transparent"
+    property color backgroundHoverColor: "transparent"
+    property color backgroundDisabledColor: "transparent"
+    property alias iconName: icon.icon
+
+    property bool showBackground: false
```

Comment:
This flag should probably also rather be encoded in a `type` similar to how it's done in `StatusButton`

Reply:
Referring to `flat` here

Reply:
Okay, no problem